portfolio/src/main/webapp/message-fetch.js

```diff
@@ -0,0 +1,24 @@
+//placeholder mock fetch object until this branch has access to the server
```

Comment:
JS Comments should have a space after the // before the text.

https://google.github.io/styleguide/jsguide.html#formatting-comments

Apply throughout